### PR TITLE
changed kubernetes setup doc

### PR DESCRIPTION
**Pull request by DerMolly**
### Component/Part
docs

### Description
This PR improves the kubernetes documentation.

### Steps

- [x] Added / updated documentation
- [x] I read the [contribution documentation](https://github.com/hedgedoc/hedgedoc/blob/master/CONTRIBUTING.md) and signed-off my commits to accept the DCO.

### CODE REVIEW

File: docs/setup/kubernetes.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Kubernetes
 
-To install use `helm install stable/hackmd`.
+HedgeDoc currently does not support any deployment via Kubernetes.
 
-For all further details, please check out the offical HedgeDoc  [K8s helm chart](https://github.com/kubernetes/charts/tree/master/stable/hackmd).
+We would love help with that, so if you want to write this, please feel free to chat us up on [Matrix](https://chat.hedgedoc.org/), so we can provide you with a repo.
```
Comment (mrdrogdrog) [inline docs/setup/kubernetes.md:5]: Well. I don't know. I would write something like.

```suggestion
If you want to help us creating a helm chart, then feel free to contact us on [Matrix](https://chat.hedgedoc.org/) or [Discourse](https://community.hedgedoc.org/).
```

**Review by mrdrogdrog — CHANGES_REQUESTED** [review]

**Review by mrdrogdrog — APPROVED** [review]

File: docs/setup/kubernetes.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Kubernetes
 
-To install use `helm install stable/hackmd`.
+HedgeDoc currently does not support any deployment via Kubernetes.
 
-For all further details, please check out the offical HedgeDoc  [K8s helm chart](https://github.com/kubernetes/charts/tree/master/stable/hackmd).
+If you want to help us creating a helm chart, then feel free to contact us on [Matrix](https://chat.hedgedoc.org/) or [Discourse](https://community.hedgedoc.org/).
+
+~~To install use `helm install stable/hackmd`.~~
```
Comment (davidmehren) [inline docs/setup/kubernetes.md:7]: I would completely remove these lines.
Why should we first write "we don't support Kubernetes" and then mention a helm chart for a different project?

**Review by davidmehren — CHANGES_REQUESTED** [review]

File: docs/setup/kubernetes.md (hunk 2 above)
Comment (DerMolly) [reply docs/setup/kubernetes.md:7]: I was not to sure if we want to remove it, so I opted to just strike through the old kubernetes documentation. But I'm more than willing to throw it out all together.

Comment (mrdrogdrog) [reply docs/setup/kubernetes.md:7]: In this case I would only remove the last line.

Comment (DerMolly) [reply docs/setup/kubernetes.md:7]: I think this would yield more confusion than be helpful

Comment (mrdrogdrog) [reply docs/setup/kubernetes.md:7]: Why?.. `We don't support kubernetes. If you want to help us then contact us.` Why should that be confusing?

Comment (DerMolly) [reply docs/setup/kubernetes.md:7]: the file has three lines and I guess you just saw the one I added and `
~~To install use 'helm install stable/hackmd'.~~`. But yes remove all the old documentation seems reasonable. Let me do that.

**Review by davidmehren — APPROVED** [review]

**Review by ErikMichelson — CHANGES_REQUESTED** [review]
If we state here that we don't support Kubernetes, we should remove it from the list of supported ways in the README as well.

**Comment by DerMolly** [comment]
Good point.

**Review by ErikMichelson — APPROVED** [review]

**Pull request merged**